invoices_app/forms.py

```diff
@@ -44,14 +46,17 @@ def is_valid(self):
         valid = super().is_valid()
         if not valid:
             return valid
+        for file_field in ['invoice_file', 'po_file']:
```

Comment:
Should these values be moved to constans in constants.py?

Reply:
+1

is a good idea to store this kind of strings in constants.